src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1587,6 +1604,18 @@ public void unpublishFeedbackSession(String feedbackSessionName,
         }
     }
     
+    public EmailWrapper sendAllLinksToNewStudentEmail(String courseId,
```

Comment:
This method name is somewhat confusing. Something like "sendFeedbackSessionsSummaryOfCourseToNewStudentEmail" is better, albeit may be a bit too long.